docs/samples/client/kfserving_sdk_sample.ipynb

```diff
@@ -138,15 +138,39 @@
     "                                 requests={'cpu':'100m','memory':'1Gi'},\n",
     "                                 limits={'cpu':'100m', 'memory':'1Gi'}))))\n",
     "\n",
-    "kfsvc = V1alpha2KFService(api_version=api_version,\n",
-    "                          kind=constants.KFSERVING_KIND,\n",
-    "                          metadata=client.V1ObjectMeta(\n",
-    "                              name='flower-sample', namespace='kubeflow'),\n",
-    "                          spec=V1alpha2KFServiceSpec(default=default_endpoint_spec,\n",
-    "                                                     canary=canary_endpoint_spec,\n",
-    "                                                     canary_traffic_percent=10))\n",
-    "\n",
-    "KFServing.patch('flower-sample', kfsvc)"
+    "KFServing.rollout_canary('flower-sample', canary=canary_endpoint_spec, percent=10, namespace='kubeflow')"
```

Comment:
looks `rollout_canary` also supports watch, so `get` step can be combined here ?

Reply:
Can you also add a comment before this line.. so it's easy to distinguish between canary definition and rollout command?

Reply:
Updated.